src-electron/db/query-command.js

```diff
@@ -1385,24 +1387,26 @@ async function selectAllCommandDetailsFromEnabledClusters(
       db,
       `
   SELECT
-    COMMAND.COMMAND_ID,
-    COMMAND.NAME,
+    CLUSTER.NAME AS CLUSTER_NAME,
+    CLUSTER.CLUSTER_ID,
     COMMAND.CODE,
-    COMMAND.SOURCE,
-    COMMAND.MANUFACTURER_CODE,
+    COMMAND.COMMAND_ID,
     COMMAND.DESCRIPTION,
-    COMMAND.RESPONSE_REF,
-    COMMAND.RESPONSE_NAME,
+    COMMAND.MANUFACTURER_CODE,
     COMMAND.MUST_USE_TIMED_INVOKE,
+    COMMAND.NAME,
+    COMMAND.RESPONSE_NAME,
+    COMMAND.RESPONSE_REF,
+    COMMAND.SOURCE,
+    ENDPOINT_TYPE_CLUSTER.ENABLED,
     ENDPOINT_TYPE_CLUSTER.SIDE,
-    CLUSTER.NAME AS CLUSTER_NAME,
-    ENDPOINT_TYPE_CLUSTER.ENABLED
+    ENDPOINT_TYPE_CLUSTER.ENDPOINT_TYPE_CLUSTER_ID
   FROM COMMAND
   INNER JOIN CLUSTER
   ON COMMAND.CLUSTER_REF = CLUSTER.CLUSTER_ID
   INNER JOIN ENDPOINT_TYPE_CLUSTER
   ON CLUSTER.CLUSTER_ID = ENDPOINT_TYPE_CLUSTER.CLUSTER_REF
-  WHERE ENDPOINT_TYPE_CLUSTER.CLUSTER_REF in (${endpointTypeClusterRef})
+  WHERE ENDPOINT_TYPE_CLUSTER.ENDPOINT_TYPE_CLUSTER_ID in (${endpointTypeClusterRef})
```

Comment:
Would be very careful with changing these. Can blow ZUC command generation.

Reply:
Why do we need this change? Would this not work with `ENDPOINT_TYPE_CLUSTER.CLUSTER_REF`. I will have to look at the schema but am just worried about this causing any regen diff issues in ZUC. Can we avoid changing this?

Reply:
`ENDPOINT_TYPE_CLUSTER.CLUSTER_REF` is the Cluster id while an endpoint specific cluster id needs to be used to compare with `endpointTypeClusterRef`